src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdNewCaseCreator.java

```diff
@@ -105,7 +111,7 @@ public CreateCaseResult createNewCase(
         } catch (UnprocessableEntity exception) {
             ClientServiceErrorResponse errorResponse = serviceResponseParser.parseResponseBody(exception);
             return new CreateCaseResult(errorResponse.warnings, errorResponse.errors);
-        // exceptions received from transformation client
+            // exceptions received from transformation client
```

Comment:
wierd that intellij did that...

Reply:
It did that, but it makes sense, as the comments are now at the same indentation level as the code

Reply:
I can remove this change if you're against it.